HomeWork/Main.java

```diff
@@ -0,0 +1,203 @@
+package HomeWork;
+
+public class Main {
+    private static Employee[] employees = new Employee[10];
+    public static void main(String[] args){
+        employees[0] = new Employee("Вася Пупкин Иванович", 1, 10000);
+        employees[1] = new Employee("Дима Пупкин Петрович", 2, 10000);
+        employees[2] = new Employee("Вася Непупкин Владимирович", 3, 11000);
+        employees[3] = new Employee("Кирил Иванов Христофорович", 4, 20000);
+        employees[4] = new Employee("Петр Первый Иванович", 5, 30000);
+        employees[5] = new Employee("Андрей Димов Кирилович", 1, 10000);
+        employees[6] = new Employee("Ирина Пупкина Ивановна", 3, 15000);
+        employees[7] = new Employee("Мария Ворникова Константиновна", 5, 19000);
+        employees[8] = new Employee("Анна Ворникова Константиновна", 5, 19000);
+        employees[9] = new Employee("Кристина Ворникова Константиновна", 5, 19000);
+        printAllEmployees();
+        System.out.println(getAllSalariesCost());
+        System.out.println(getMinSalary());
+        System.out.println(getMediumSalariesCost());
+        getNames();
+        getLessSalary(12500);
+        getMoreSalary(11000);
+        indexAllSalaries(3);
+        indexDepartmentSalaries(2,4);
+        System.out.println(getAllSalariesCost());
+        System.out.println(getMaxSalary());
+        System.out.println(getMediumSalariesCost());
+        getNames();
+        System.out.println(getDepartmentMinSalary(5));
+        System.out.println(getDepartmentMaxSalary(5));
+        System.out.println(getDepartmentSumSalary(5));
+        System.out.println(getDepartmentAverageSalary(5));
+        printAllEmployeesDepartment(5);
+
+        }
+
+    public static void printAllEmployees() {
+        for (Employee employee : employees) {
+            System.out.println(employee);
+        }
+    }
+    public static double getAllSalariesCost() {
+        double totalSalaryCost = 0.0;
+        for (Employee employee : employees) {
+            if (employee != null) {
+                totalSalaryCost += employee.getEmployeeSalary();
+            }
+        }
+        return totalSalaryCost;
+    }
+
+    public static Employee getMinSalary() {
+        Employee employeeWithMinSalary = null;
+        double minSalary = Integer.MAX_VALUE;
+        for (Employee employee : employees) {
+            if (employee != null) {
+                if (minSalary > employee.getEmployeeSalary()) {
```

Comment:
вложенные if можно заменить на один через И, т.е.
```suggestion
                if (employee != null && minSalary > employee.getEmployeeSalary()) {
```